frontend/mobile/pubspec.yaml

```diff
@@ -37,6 +37,7 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
   dio: ^5.3.3
+  easy_search_bar: ^2.5.0
```

Comment:
this is needed ? You didn't use it in the code

Reply:
didn't even know I had this